client/lib/wp/offline-library/index.js

```diff
@@ -0,0 +1,102 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import stringify from 'json-stable-stringify';
+
+const readCache = () => {
+	try {
+		return require( '../../../../cached-requests.json' );
```

Comment:
can this just be `require( 'cached-requests.json' )`

Reply:
oh that's probably right!

Reply:
de-relatived in 9107268